projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-options/annotation-options.component.spec.ts

```diff
@@ -19,10 +19,10 @@ const mockColorService = {
   providers: [{ provide: NG_VALUE_ACCESSOR, multi: true, useExisting: forwardRef(() => MockColorPickerComponent) }],
 })
 class MockColorPickerComponent implements ControlValueAccessor {
-  writeValue(obj: any): void {}
-  registerOnChange(fn: any): void {}
-  registerOnTouched(fn: any): void {}
-  setDisabledState?(isDisabled: boolean): void {}
+  writeValue(obj: any): void { /* TODO document why this method 'writeValue' is empty */ }
+  registerOnChange(fn: any): void { /* TODO document why this method 'registerOnChange' is empty */ }
+  registerOnTouched(fn: any): void { /* TODO document why this method 'registerOnTouched' is empty */ }
+  setDisabledState?(isDisabled: boolean): void { /* TODO document why this method 'setDisabledState' is empty */ }
```

Comment:
If we really need to document why the mock has empty methods, please do so. I don't think we need TODOs for that.

Reply:
sonarcloud show a critical error for an empty method, so added a comments for resolving sonarcloud issue